Edits and clarification based on customer feedback (#11770)
MicrosoftDocs/windows-itpro-docs: windows/deployment/do/mcc-enterprise-deploy.md

   :::image type="content" source="./images/use-custom-dns-3.png" alt-text="Screenshot of multiple installer questions about setting an alternate DNS server." lightbox="./images/use-custom-dns-3.png":::
   If you use a dynamic IP address, the DHCP server will automatically configure the IP address and DNS settings. 
 
1. Choose where you would like to download, install, and store the virtual hard disk for EFLOW. You'll also be asked how much memory, storage, and how many cores you would like to allocate for the VM. For this example, we chose the default values for download path, install path, and virtual hard disk path. 
   <!-- Insert Image 4 -->
   :::image type="content" source="./images/installation-info-4.png" alt-text="Screenshot of multiple installer questions about memory and storage." lightbox="./images/installation-info-4.png":::
   Please refer to the [Sizing Recommendations](/mcc-enterprise-prerequisites#sizing-recommendations) for memory, virtual storage, and CPU cores. For this example we chose the recommend values for a Branch Office/Small Enterprise deployment.
   <!-- Insert Image 5 -->

>    :::image type="content" source="./images/memory-storage-5.png" alt-text="Screenshot of multiple installer questions about setting an alternate DNS server." lightbox="./images/memory-storage-5.png":::

Reviewer: ```suggestion
   :::image type="content" source="./images/memory-storage-5.png" alt-text="Screenshot of multiple installer questions about memory and storage." lightbox="./images/memory-storage-5.png":::
```

   <!-- Remove: If this is your first MCC deployment, select **n** so that a new IoT Hub can be created. If you have already configured MCC before, choose **y** so that your MCCs are grouped in the same IoT Hub.

    1. You'll be shown a list of existing IoT Hubs in your Azure subscription. Enter the number corresponding to the IoT Hub to select it. **You'll likely have only 1 IoT Hub in your subscription, in which case you want to enter "1"**

       :::image type="content" source="./images/ent-mcc-script-select-hub.png" alt-text="Screenshot of the installer script running in PowerShell prompting you to select which IoT Hub to use." lightbox="./images/ent-mcc-script-select-hub.png":::
       -->
1.  When the installation is complete, you should see the following output (the values below will be your own)
       :::image type="content" source="./images/ent-mcc-script-complete.png" alt-text="Screenshot of the installer script displaying the completion summary in PowerShell." lightbox="./images/ent-mcc-script-complete.png":::
       <!-- Insert Image 7 -->

>     :::image type="content" source="./images/installation-complete-7" alt-text="Screenshot of expected output when installation is complete." lightbox="./images/installation-complete-7":::

Reviewer: ```suggestion
    :::image type="content" source="./images/installation-complete-7.png" alt-text="Screenshot of expected output when installation is complete." lightbox="./images/installation-complete-7.png":::
```
